packages/foundation-models/src/azure-openai/azure-openai-chat-client.test.ts

```diff
@@ -134,9 +134,9 @@ describe('Azure OpenAI chat client', () => {
 
     mockDeploymentsList(
       {
-        scenarioId: 'foundation-models',
+        scenarioId: 'azure-openai',
         resourceGroup: 'custom-resource-group',
-        executableId: 'azure-openai'
+        executableId: 'foundation-models'
```

Comment:
[q] Isn't this the other way around? scenarioId is 'foundation-models' and executableId changes depending on the model: https://help.sap.com/docs/ai-launchpad/sap-ai-launchpad/models-and-scenarios-in-generative-ai-hub-fef463b24bff4f44a33e98bb1e4f3148
@ZhongpinWang

Reply:
I suspect this is the reason why some of the tests are failing.